models/lorenz_96/work/input.nml

```diff
@@ -92,9 +92,12 @@
 &ensemble_manager_nml
    /
 
+# the localization cutoff in this file is distributed is too
+# large for the model to converge.  to test that the model is 
+# doing a successful assimilation, change cutoff to 0.02 and rerun.
```

Comment:
```suggestion

```

Reply:
if this text is still somewhere (maybe in workshop setup?) it should be 'as distributed is ...'   

if it's completely removed, never mind!